src/main/resources/alma/fix/item.fix

```diff
@@ -55,44 +61,49 @@ do list(path: "HOL  ", "var": "$i")
 end
 
 do list(path:"POR  ", "var": "$i")
-  copy_field("$i.M","@HOL-M_POR-M.$append")
-# entity for every POR  .a without POR  .A
-  unless any_match("$i.a",".*6441$") # filter out hbz
-    set_hash( "hasItem[].$append")
-    set_array("hasItem[].$last.type[]", "Item", "DigitalDocument")
-    add_field("hasItem[].$last.label", "Electronic Portfolio")
-    copy_field("$i.D", "$i.@electronicLocator")
-    replace_all("$i.@electronicLocator","https://eu04.alma.exlibrisgroup.com/view/uresolver/49HBZ_NETWORK","")
-    paste("hasItem[].$last.electronicLocator", "~https://eu04.alma.exlibrisgroup.com/view/uresolver/","$i.M","$i.@electronicLocator", join_char: "")
-    copy_field("$i.d", "$i.@sublocation")
-    replace_all("$i.@sublocation","https://hbz-network.userservices.exlibrisgroup.com/view/uresolver/49HBZ_NETWORK","")
-    paste("hasItem[].$last.sublocation", "~https://hbz-network.userservices.exlibrisgroup.com/view/uresolver/","$i.M","$i.@sublocation", join_char: "")
-    call_macro("holdingLibrary+opacLink", mmsIdField:"$i.a",libraryCodeField:"$i.g")
-  end
-  # entity for every POR  .A
-  if exists ("$i.A")
-    copy_field("$i.D", "$i.@electronicLocator")
-    replace_all("$i.@electronicLocator","https://eu04.alma.exlibrisgroup.com/view/uresolver/49HBZ_NETWORK","")
-    copy_field("$i.d", "$i.@sublocation")
-    replace_all("$i.@sublocation","https://hbz-network.userservices.exlibrisgroup.com/view/uresolver/49HBZ_NETWORK","")
-    do list(path:"$i.A", "var": "$j")
+  if none_equal("$i.b", "Not Available")
```

Comment:
This conditional introduces something new. A suppression mechanism that was not there before.

Reply:
Neither were the other two. But you would prefer one commit with the holding/item changes (no test coverage) and one with the portfolio changes?

Reply:
Ah, I think I understand. Sorry for being dense so early in the morning ;) You mean the suppression of the portfolio, not the suppression of the record; holdings and items were suppressed before, but the portfolio wasn't.